bayesian/generate_manifest.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from lxml.etree import SubElement, Element, ElementTree
+from io import BytesIO
+
+
+class PomXMLTemplate:
+
+    def __init__(self, json_data=None):
+        if json_data:
+            self._data = dict(json_data)
+        else:
+            raise ValueError
+        self.root = Element(
+            'project',
+            xmlns="http://maven.apache.org/POM/4.0.0",
+        )
+        self.tree = ElementTree(self.root)
+        self.create()
+
+    def create(self):
+        self._project = self._data.get('project', None)
```

Comment:
The `get` method defaults to None if the given key is present, so no need to pass `None` (here and bellow as well):
```python
>>> {}.get('foo') is None
True
```